articles/sentinel/automate-incident-handling-with-automation-rules.md

```diff
@@ -127,12 +127,35 @@ In order for an automation rule to run a playbook, this account must be granted
 
 When you're configuring an automation rule and adding a **run playbook** action, a drop-down list of playbooks will appear. Playbooks to which Azure Sentinel does not have permissions will show as unavailable ("grayed out"). You can grant Azure Sentinel permission to the playbooks' resource groups on the spot by selecting the **Manage playbook permissions** link.
 
-> [!NOTE]
-> **Permissions in a multi-tenant architecture**
->
-> Automation rules fully support cross-workspace and multi-tenant deployments (in the case of multi-tenant, using [Azure Lighthouse](extend-sentinel-across-workspaces-tenants.md#managing-workspaces-across-tenants-using-azure-lighthouse)).
->
-> Therefore, if your Azure Sentinel deployment uses a multi-tenant architecture (If you're an MSSP, for example), you can have an automation rule in one tenant run a playbook that lives in a different tenant, but permissions for Sentinel to run the playbooks must be defined in the tenant where the playbooks reside, not in the tenant where the automation rules are defined.
+#### Permissions in a multi-tenant architecture
+
+Automation rules support multi-tenant scenarios (using [Azure Lighthouse](https://docs.microsoft.com/azure/lighthouse/)), where a service provider tenant manages an Azure Sentinel workspace in a customer tenant.
+
+There are two main scenarios when managing cross-tenant automation rules:
+
+- Automation rule created in the customer tenant is configured to run a playbook located in the service provider tenant. This approach is normally used to protect intellectual property in the playbook. Nothing special is required for this scenario to work. Just grant permissions to the relevant resource group where the playbook is located via **Manage playbook permissions** menu as explained [here](tutorial-respond-threats-playbook.md#respond-to-incidents).
+
+- Automation rule created in the customer tenant is configured to run a playbook located in the customer tenant. Used when there is no need to protect intellectual property. For this scenario to work, permissions to execute the playbook need to be granted to Azure Sentinel in ***both tenants***. In the customer tenant, you grant them via **Manage playbook permissions** menu as explained [here](tutorial-respond-threats-playbook.md#respond-to-incidents). To grant the relevant permissions to the service provider tenant, you need to include the **Azure Security Insights** app in your Azure Lighthouse delegation template with the **Azure Sentinel Automation Contributor** role. The scenario looks like this:
+
+:::image type="content" source="media/automate-incident-handling-with-automation-rules/automation-rule-multi-tenant.png" alt-text="Automation Rules multi-tenant":::
+
+In order to add the Azure Security Insights app in your Azure Lighthouse onboarding template, you will need to find its Object ID. To find this:
```

Comment:
From here to the end of this addition belongs in [this document](tutorial-respond-threats-playbook.md), probably in the Respond to incidents section, as a third bullet "In a service provider deployment" here:
![image](https://user-images.githubusercontent.com/61194355/120323798-90aa0a80-c2ee-11eb-9408-88710d19b818.png)

Reply:
agree, how do I make the change?